packages/babel-traverse/src/path/modification.ts

```diff
@@ -279,7 +279,7 @@ export function updateSiblingKeys(
 ) {
   if (!this.parent) return;
 
-  const paths = pathCache.get(this.parent);
+  const paths = getCachedPaths(this.hub, this.parent) ?? ([] as never[]);
```

Comment:
@liuxingbaoyu We already use `??` in many places -- I'll avoid it here since in this case the compiled output uses a temp variable. Although, I'm not too worried about these small differences in our compiled code, since it's usually not shipped to the browser.
```suggestion
  const paths = getCachedPaths(this.hub, this.parent) || ([] as never[]);
```